src/js/index.js

```diff
@@ -749,3 +793,15 @@ function switchTo(element) {
 	reloadMap(user_selected_value);
 	custom_select.select(".select-items").remove();
 }
+
+function switchToNetwork(element) {
+	const custom_select_network = d3.select("#custom-select-network");
+	custom_select_network.select("select").property("value", element.id.substring(12));
```

Comment:
why are we `substring`'ing here?

Reply:
To get the value and update when it's selected @KhadijaMahanga